src/app/app.component.ts

```diff
@@ -77,6 +77,7 @@ export class AppComponent implements OnInit {
   }
 
   private logOutExpiredUser(): void {
```

Comment:
This probably should already be handled in app.component.ts when listening to navigation changes.